### PR TITLE
Create account card

### PR DESCRIPTION
**Pull request by tintinthong**

### CODE REVIEW

File: packages/experiments-realm/components/avatar-group.gts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class AvatarGroup extends GlimmerComponent<AvatarGroupSignature>
       }
       .avatar-thumbnail {
         flex-shrink: 0;
-        --profile-avatar-icon-size: 60px;
+        --profile-avatar-icon-size: var(--boxel-avatar-group-icon-size, 60px);
```
Comment (lucaslyl) [inline packages/experiments-realm/components/avatar-group.gts]: remove --boxel-avatar-group-icon-size

File: packages/experiments-realm/crm/contact.gts [hunk 2]
```diff
@@ -584,20 +584,42 @@ class FittedTemplate extends Component<typeof Contact> {
...
-  @field phoneOffice = contains(PhoneField); // Stores office phone, format may change after final phoneField PR is approved
-  @field status = contains(StatusField); // Stores status as a status field
-  //@field socialLinks = containsMany(UrlField); // Pending format discussion with Burcu for consistency
-  //@field account = linksTo(Account) // Pending completion of account card
+  @field primaryEmail = contains(StringField);
+  @field secondaryEmail = contains(StringField);
+  @field phoneMobile = contains(PhoneField);
+  @field phoneOffice = contains(PhoneField);
+  @field status = contains(StatusField);
+  //@field socialLinks = containsMany(UrlField);
```
Comment (lucaslyl) [inline packages/experiments-realm/crm/contact.gts]: add Burcu contact links field

File: packages/experiments-realm/website.gts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import WorldWwwIcon from '@cardstack/boxel-icons/world-www';
```
Comment (lucaslyl) [inline packages/experiments-realm/website.gts:1]: use entity display components

**Comment by tintinthong** [comment]
Thx for the integration lucas. State of this PR is that some of the fitted template in contact is running. WIll fix in another PR.

**Pull request merged**